src/Parser.js

```diff
@@ -392,52 +386,47 @@ var styleFuncs = [
  *   small text {\Large large text} small text again
  * It is also used for \left and \right to get the correct grouping.
  *
- * @return {?ParseResult}
+ * @return {?ParseNode}
  */
-Parser.prototype.parseImplicitGroup = function(pos) {
-    var start = this.parseSymbol(pos);
+Parser.prototype.parseImplicitGroup = function() {
+    var start = this.parseSymbol();
 
-    if (!start || !start.result) {
+    if (start === null) {
```

Comment:
At KA we prefer using `==` (or `!=`) when comparing against `null` because it will produce the same results if `start` is `undefined`.  In this case it's pretty easy to verify that `start` is either a `ParseNode` or `null` so it's not a big deal, but in other cases it may not be easy to make this determination.